### DISCUSSION TITLE
Need help I’m new to this

### DISCUSSION THREAD

**Discussion by Stopjia4sho** [nodeschool/discussions#2648] [General]
Please can someone help me someone is using this program to control my phone

**Comment by Kawoo1** [comment]
Hi Stopjia4sho! Im surprised no ones responded to your discussion post yet. But for Github theres lots of tutorials onlinw for navigating on both browser and ios/android. For Node and Node.js there are equally great places to get started. I guess your post was too broad for specific answers, I would love to assist if you still needed it though.
